src/components/NewTab/NewTab.tsx

```diff
@@ -10,12 +10,48 @@ import Twitter from "../../assets/icons/Twitter"
 import Web from "../../assets/icons/Web"
 import ConfigPanel from "../ConfigPanel/ConfigPanel"
 import Cog from "../../assets/icons/Cog"
+import Eye from "../../assets/icons/Eye"
+import Resize from "../../assets/icons/Resize"
 
 export interface ConfigList {
     datetime: boolean;
     datetimePosition: string;
     hours12: boolean;
     color: string;
+    bgcolor: string;
+    blurBool: boolean;
+    blur: number;
+    opacityBool: boolean;
+    opacity: number;
+    shadowBool: boolean;
+}
+
+let isViewClean = false
+
+// Toggle the distraction free view
+function toggleCleanView(force: boolean) {
+    // Disable various settings
+    if (isViewClean) {
+        document.querySelector('.new-tab').classList.remove('cleaned');
+        document.querySelector('.group-eye')?.classList.remove('active');
+    } else {
+        document.querySelector('.new-tab').classList.add('cleaned');
+        document.querySelector('.group-eye')?.classList.add('active');
+    }
+
+    // Check if datetime exists
+    if (document.querySelector('.datetime')) {
+        // Hide datetime
+        document.querySelector('.datetime').style.opacity = isViewClean ? 1 : 0;
+    }
+
+
+    // set clean state
+    isViewClean = isViewClean ? false : true;
+}
+
+function toggleFullView() {
+    document.querySelector('.new-tab-image')?.classList.toggle('fit');
 }
```

Comment:
About this vanilla JS block, i changed it to a reactive value
```ts
const [isFullView, setIsFullView] = useState(false)
const [isCleanView, setIsCleanView] = useState(false)
```
```tsx
<div className={`new-tab ${isCleanView ? "cleaned" : ""}`}>
<img className={`new-tab-image ${isFullView ? 'fit' : ''}`} />
```
So instead of calling the `toggleCleanView()` it calls `setIsCleanView((prev) => !prev)` and so the html re-renders with the right classes